package.json

```diff
@@ -5,7 +5,6 @@
   "private": true,
   "scripts": {
     "start": "electron-webpack dev",
-    "postinstall": "npm run generate-licenses-ts",
```

Comment:
I think this was removed when we were trying to get it to run at the beginning of the year - should I put that line back in?

Reply:
Yes. We do need this line to generate the right list of licenses in the CI. This will only affect `npm install` so if you don't plan to run it in a while, it's safe to have it back in. Or just remove it but take care not to commit it. I will see if `postinstall` is still not working. If that's the case, I will open an issue to look into that.